execution.bs

```diff
@@ -4960,9 +4960,124 @@ enum class forward_progress_guarantee {
             except as noted below:
 
               <pre highlight="c++">
-              template &lt;class Tag, class Data, class... Child> // arguments are not associated entities ([lib.tmpl-heads])
-              struct <i>basic-sender</i> : <i>unspecified</i> {
-                using is_sender = <i>unspecified</i>;
+              template&lt;class T, class... Us>
+              concept <i>one-of</i> = (same_as&lt;T, Us> ||...); // exposition only
+
+              template&lt;template&lt;class...> class T, class... Args>
+              concept <i>well-formed</i> = requires { typename T&lt;Args...>; }; // exposition only
+
+              template&lt;const auto& Fun, class... Args>
+              concept <i>cpo-callable</i> = <i>callable</i>&lt;decltype(Fun), Args...>; // exposition only
```

Comment:
is this actually invocable - or is there a callable elsewhere?  or are we trying to channel this? http://eel.is/c++draft/func.def#4

Reply:
got it -- thanks.